dom-renderer.jsx

```diff
@@ -9,7 +9,7 @@ const cubicBezier = easing(0, 1, 0, 1);
 const physicsScene = physicsManager.getScene();
 
 class DomItem extends THREE.Object3D {
-  constructor(position, quaternion, scale, width, height, worldWidth, render) {
+  constructor(position, quaternion, scale, width, height, worldWidth, render, domRenderEngine) {

```

Comment:
domRenderer is the same thing, you should access the singleton, no need to pass the reference in here